content/en/docs/reference/kubernetes-api/ref/common-definitions/nodeselectorrequirement-.md

```diff
@@ -0,0 +1,39 @@
+---
+api_metadata:
+  apiVersion: ""
```

Comment:
nit: Some of the file names could be hyphenated. For example:
node-selector-requirement-v1.md?

Reply:
not in this case, @kbhawkey 
'NodeSelectorRequirement' is the name of a definition, though not exposed as a resource.

Reply:
OK. If the metadata is not needed then perhaps it should not get added to this type of file.